coverage-tests.js

```diff
@@ -2440,4 +2440,41 @@ test('should capture webview when specified in check settings on android', {
   },
 })
 
+test('should support removal of duplicate test results', {
+  page: 'Default',
+  vg: true,
+  config: {
+    removeDuplicateTestsPerBatch: true,
+  },
+  test({eyes, assert}) {
+    eyes.open({appName: 'Applitools Eyes SDK'})
+    eyes.check({isFully: false})
+    eyes.close(false)
+    eyes.open({appName: 'Applitools Eyes SDK'})
+    eyes.check({isFully: false})
+    eyes.close(false)
+    const results = eyes.runner.getAllTestResults(false)
+    assert.equal(results.getAllResults().length, 1)
+  },
+})
+
+test('should skip removal of duplicate test results when baseline name used', {
+  page: 'Default',
+  vg: true,
+  config: {
+    removeDuplicateTestsPerBatch: true,
```

Comment:
It should not go to the configuration.

Reply:
@kyrylodolynskyi and I talked about this offline. Keeping it in config for now is OK. In the future we might want to introduce a `runner` option to tests.